sam-per-ref-count/bin/sam-per-ref-count.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+import pysam
+import argparse
+
+parser = argparse.ArgumentParser(description='Count reads from sam/bam file')
+parser.add_argument("-i", "--ifile", help="Seq file name with read information as input in sam/bam format ")
+parser.add_argument("-f", "--fileformat", default="bam", help="File format of input file - either sam or bam; default is for bam format (rb)")
+parser.add_argument("-c", "--ref-col-name", default="reference", help="Header of column wih references, default: reference")
+parser.add_argument("-d", "--cnt-col-name", default="count", help="Header of column with read count per reference, default")
```

Comment:
"Name of output columns with read count"

Also the description is missing the default.